service/init.go

```diff
@@ -318,6 +319,11 @@ func (smf *SMF) Start() {
 	//Send NRF Registration
 	smf.SendNrfRegistration()
 
+	if smfCtxt.EnableNrfCaching {
+		initLog.Infoln("Enable NRF caching feature")
```

Comment:
Probably, print cache eviction time from config also in this log

Reply:
Done.